engine/src/flutter/impeller/renderer/backend/gles/test/reactor_unittests.cc

```diff
@@ -96,33 +96,6 @@ TEST(ReactorGLES, UntrackedHandle) {
   EXPECT_TRUE(reactor->React());
 }
 
-TEST(ReactorGLES, NameUntrackedHandle) {
```

Comment:
Instead of deleting this, why not just put it in the preprocessor guard.  This test seems to point to a better way to disable this: `SupportsDebugLabels()`. If we turn that off this test will be skipped automatically.

Reply:
Yeah i shouldn't remove this.

Reply:
The gtest skip still triggers the gmock assertions though, is there a way to disable them?

Reply:
Can you just move the expect calls to after the skip?

Reply:
Its after we've moved the  mock gles :(

Reply:
Are you sure the GTEST_SKIP doesn't just handle cancelling the expects?

You could keep a raw pointer around to do the expect_calls after the skips.  It's not pretty but it's a test and we know it will still be in memory.

Reply:
It does not :(

Reply:
I'll try the raw_ptr